src/components/SystemsViewTable/SystemsViewTable.tsx

```diff
@@ -0,0 +1,160 @@
+import React, { useState } from 'react';
+import {
+  DataView,
+  DataViewTextFilter,
+  useDataViewPagination,
+} from '@patternfly/react-data-view';
+import {
+  DataViewTable,
+  DataViewTh,
+  DataViewTr,
+} from '@patternfly/react-data-view/dist/dynamic/DataViewTable';
+import { useDataViewSelection } from '@patternfly/react-data-view/dist/dynamic/Hooks';
+import { Bullseye, Pagination, Spinner, Tooltip } from '@patternfly/react-core';
+import DisplayName from '../../routes/Systems/components/SystemsTable/components/columns/DisplayName';
+import Workspace from '../../routes/Systems/components/SystemsTable/components/columns/Workspace';
+import Tags from '../../routes/Systems/components/SystemsTable/components/columns/Tags';
+import OperatingSystem from '../../routes/Systems/components/SystemsTable/components/columns/OperatingSystem';
+import LastSeen from '../../routes/Systems/components/SystemsTable/components/columns/LastSeen';
+import { DataViewToolbar } from '@patternfly/react-data-view/dist/dynamic/DataViewToolbar';
+import DataViewFilters from '@patternfly/react-data-view/dist/dynamic/DataViewFilters';
+import {
+  ResponsiveAction,
+  ResponsiveActions,
+  BulkSelect,
+} from '@patternfly/react-component-groups';
+import { useSystemsQuery } from './hooks/useSystemsQuery';
+
+const PER_PAGE = 50;
+const INITIAL_PAGE = 1;
+
+const SystemsViewTable: React.FC = () => {
+  const pagination = useDataViewPagination({
+    perPage: PER_PAGE,
+    page: INITIAL_PAGE,
+  });
+
+  const selection = useDataViewSelection({
+    matchOption: (a, b) => a.id === b.id,
+  });
+
+  const { data, total, isLoading } = useSystemsQuery({
+    page: pagination.page,
+    perPage: pagination.perPage,
+  });
+
+  if (isLoading || !data) {
```

Comment:
I think [DataView](https://www.patternfly.org/extensions/data-view/overview/#dataview) has a loading state itself.
I think it's `activeState` prop, but I'm not sure how it works (and if correctly)